cmd/logsPipelineJob.go

```diff
@@ -160,6 +163,20 @@ func getLogsJob(cmd *cobra.Command, apiClient *apiclient.Radixapi, appName, jobN
 	}
 }
 
+func isCompletedJob(status string) bool {
+	return slices.Contains(completedJobStatuses, status)
+}
+
+func errorAndLogJobStatus(status string, cmd *cobra.Command) error {
+	msg := fmt.Sprintf("job completed with status %s", status)
+	if status == jobStatusFailed {
+		fmt.Fprintln(cmd.OutOrStdout())
+		return errors.New(msg)
+	}
+	log.Print(cmd, "radix-cli", msg, log.Red)
```

Comment:
Kan vi printe ut "success" meldinger i Grønt istedenfor Rødt?

Reply:
Endret til å bruke logrus for å printe info